### PR TITLE
Mandatory Item 2: Queue and Email service integration tested and merge conflict resolved

### PR DESCRIPTION
**Pull request by dip06ece**
Mandatory Item 2 complete.

### CODE REVIEW

File: src/main/java/com/tg5/service/MessageProducer.java [hunk 1]
```diff
@@ -0,0 +1,16 @@
+package com.tg5.service;
```
Comment (vytaux) [inline src/main/java/com/tg5/service/MessageProducer.java]: You can remove this file and inject jmsTemplate straight to LowBalanceCronAlert  and use it there.

File: src/main/java/com/tg5/Application.java [hunk 2]
```diff
@@ -1,11 +1,17 @@
...
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.scheduling.annotation.EnableScheduling;
 
 @SpringBootApplication(scanBasePackages = {"edu.miu.common", "com.tg5"})
+@EnableScheduling
 public class Application {
 
+	@Autowired
```
Comment (vytaux) [inline src/main/java/com/tg5/Application.java]: u sure you need this here?

File: src/main/java/com/tg5/service/LowBalanceCronAlert.java [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+    @Autowired
+    private AccountRepository accountRepository;
+
+    @Autowired
+    private MessageProducer messageProducer;
+
+    @Scheduled(fixedRate = 50000) // Execute every 5 seconds
+    public void sendLowBalanceAlertMessage() {
+        System.out.println("Checking all low balance accounts and sending mails.......");
+//        messageProducer.sendMessage("ams",
```
Comment (vytaux) [inline src/main/java/com/tg5/service/LowBalanceCronAlert.java]: if this is unnecessary testing you can remove it

**Review by vytaux — CHANGES_REQUESTED** [review]

File: src/main/java/com/tg5/Application.java (hunk 2 above)
Comment (dip06ece) [reply src/main/java/com/tg5/Application.java]: We need this for creating the consumer Bean.

File: src/main/java/com/tg5/service/LowBalanceCronAlert.java (hunk 3 above)
Comment (dip06ece) [reply src/main/java/com/tg5/service/LowBalanceCronAlert.java]: Discussed it: It is just for demo!

File: src/main/java/com/tg5/service/MessageProducer.java (hunk 1 above)
Comment (dip06ece) [reply src/main/java/com/tg5/service/MessageProducer.java]: Removed!

**Review by vytaux — APPROVED** [review]

**Pull request merged**